native/core/src/execution/shuffle/row.rs

```diff
@@ -3358,7 +3359,9 @@ pub fn process_sorted_row_partition(
 
         // we do not collect metrics in Native_writeSortedFileNative
         let ipc_time = Time::default();
-        written += write_ipc_compressed(&batch, &mut cursor, &ipc_time)?;
+        // compression codec is not configurable for CometBypassMergeSortShuffleWriter
```

Comment:
Did you mean TODO by this comment?

Reply:
This PR only makes compression configurable for native shuffle and not for our JVM columnar shuffle. I updated the PR title to make this clearer.